localbuild/meta.yaml

```diff
@@ -77,7 +77,7 @@ requirements:
     - PyMySQL >=0.9.3
     - validate_email
     - multidict
-    - pint <=0.22
+    - pint
```

Comment:
Have you checked the solution with an older pint version?

Reply:
Yes, it works fine (i.e. the test suite passes) with 0.22 as well.

What I am not quite sure about is if this matches the exact behavior pint previously had, or if there is some difference. But to me it seemed to be the most intuitive thing to do.